config.ts

```diff
@@ -7,7 +7,7 @@ export = {
 		{
 			username: 'guest',
 			password: ' ',
-			permissions: ['resin.device_type.get'],
+			permissions: ['resin.device_type.get', 'resin.cpu_architecture.get'],
```

Comment:
This needs to change to
```suggestion
			permissions: ['resin.device_type.read', 'resin.cpu_architecture.read'],
```
in order to continue working after the rebase